core-web/apps/dotcms-ui/src/app/view/components/dot-listing-data-table/dot-listing-data-table.component.ts

```diff
@@ -77,14 +77,11 @@ export class DotListingDataTableComponent implements OnInit {
     isContentFiltered = false;
     dateColumns: DataTableColumn[];
     loading = true;
-    contextMenuItems: MenuItem[];
+    contextMenuItems: CustomMenuItem[];
```

Comment:
Why we need a custom if we passing this to a primeng component?

Reply:
Fixed!